src/components/ListItem.jsx

```diff
@@ -69,29 +72,51 @@ export function ListItem({
 	}, [isChecked, itemId, listPath]);
 
 	const urgency = calculateUrgency(daysUntilNextPurchase, dateLastPurchased);
+	const label = { inputProps: { 'aria-label': `${name}` } };
+	const color =
```

Comment:
nested ternarys, even with identation, are problematic. This would be better with an if or a switch

edit: this doesn't look used?

Reply:
Hi Lyle! It's being used in the Checkbox component to provide the label prop!

Reply:
Hi Lyle! Oops, I was looking at the wrong code. But we fixed this in the other branch for our other PR (Issue#20)! Should fix the issue mentioned in the comment below